src/pages/Registration.jsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import { Link, useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { RegistrationSchema } from '../validation/Registration';
+import axios from '../api/axios';
+import { urls, publicLinks } from '../constants/links';
+
+function Registration() {
+  const navigate = useNavigate();
+  return (
+    <div className="w-full flex justify-center p-10 bg-gray-300 h-screen">
+      <Formik
```

Comment:
write descriptive names "formik ni"

Reply:
That is how it should be. It is a package to help in form validation.